### PR TITLE
stops the bottom of the screen from flashing on hover of last element & uses cdn for static example

### PR DESCRIPTION
**Pull request by cupOJoseph**

### CODE REVIEW

**Comment by cupOJoseph** [comment]
#73

File: example/index.html [hunk 1]
```diff
@@ -34,14 +34,11 @@
     <div class='ethAddress'>
       <threebox-address data-address='0xa8ee0babe72cd9a80ae45dd74cd3eae7a82fd5d1' data-display='full'></threebox-address>
     </div>
+    
+    <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
 
-    <div class='ethAddress'>
-      <threebox-address data-address='0xa8ee0babe72cd9a80ae45dd74cd3eae7a82fd5d1' data-theme='none'>
-        0xa8ee0babe72cd9a80ae45dd74cd3eae7a82fd5d1
-      </threebox-address>
```
Comment (oed) [inline example/index.html]: How come you removed this example?

**Review by oed — COMMENTED** [review]
Thanks @jschiarizzi!
Just have one question below:

**Comment by cupOJoseph** [comment]
Must have been a mistake, I put it back.

With the bottom few links sometimes the hover appears below the the element, pushing the screens size down, then moving the mouse off. That makes it flash appear/disappear really quickly so the extra space at the bottom is an easy static way to stop that without adding any css files

**Comment by oed** [comment]
Thanks @jschiarizzi Would you mind rebasing on master? After that I can merge this!

**Comment by oed** [comment]
Ping @jschiarizzi There is still a conflict in this PR. Once it's fixed I'm happy to merge it.

**Comment by cupOJoseph** [comment]
@oed conflict removed, it was just an empty new line causing that

**Comment by oed** [comment]
Thanks @jschiarizzi

**Pull request merged**